qtodotxt/lib/file.py

```diff
@@ -130,6 +130,15 @@ def getAllProjects(self, return_completed=False):
                         projects[project] = count
         return projects
 
+    def getAllPriorities(self, return_completed=False):
+
+        priorities = {"A": 0, "B": 0, "C": 0, "D": 0}
+        for task in self.tasks:
+            if return_completed or not task.is_complete:
+                if task.priority in priorities.keys():
+                    priorities[task.priority] += 1
+        return priorities
```

Comment:
why not:

    priorities = {}
    for task in self.tasks:
        ....
        if task.priority in priorities:
             priotities[task.priority] += 1
         else:
              prioritiies[task.priority] = 1
     return priorities

then we only show in tree existing priorities, as we do for projects etc ...